backend/backend.py

```diff
@@ -172,7 +172,7 @@ def send_recording_status(self) -> None:
                 return
         else:
             message["status"] = "off"
-        _, _, free_disk_space = disk_usage(__file__)
+        _, _, free_disk_space = disk_usage(Path(__file__).parents[0])
```

Comment:
`Path.parent` will give the direct parent.

Reply:
Done
d702a48